nss-day-cohort-44/nutshell-labyrinth-leprechauns
# Sl event edit

# Description
The events now populate with an edit btn. When you click it the entry changes to a from that can be edited. The save btn updates the api and the events are rerendered. 
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
- [] Bug fix (non-breaking change which fixes an issue)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
# Testing Instructions

create an event and click edit. a form should appear. Make some changes and save them. check to see if the api updated as well.  

# Checklist:
